makefile

```diff
@@ -65,7 +65,7 @@ get-pmids:
 
 # SETUP / INSTALLATION -------------------------------------------------------------------------------------------------
 install:
-	pip install -r requirements-unlocked.txt
+	pip install -r requirements-unlocked.txt --user --break-system-packages
```

Comment:
[Build passing now](https://github.com/monarch-initiative/omim/actions/runs/11674781775) after adding this.

Reply:
Why is `--user --break-system-packages` needed? I thought that was just to add packages that aren't in ODK and ODK should now have everything this repo needs, right?

Reply:
Here is [the rationale](https://github.com/monarch-initiative/omim/issues/114#issuecomment-2453575864). I'd rather not have to keep adding/removing `make install` every time I introduce a new package.

Reply:
Hmm.... ok. Is it mentioned somewhere, other than a PR comment, that any new non-ODK package needed here does need an ODK issue to request it :)

Reply:
I do have it in my 'rules', which I review every now and then, that I personally should open up a GH issue in such an event, but I wouldn't know where to write these things down; maybe some `mondo-ingest` developer documentation.

Also alternatively, if we wanted to make this `make install` thing more efficient, we could have it run a script that instead  iterates over each package in the `requirements.txt`, checks first if it is already installed globally, and then only do the `pip install PACKAGE --user --break-system-packages` only if it is not. Feels a little excessive, but in a way that is the most elegant approach. Right now it's not a major issue.